<launch>
  <node pkg="fixed_decoder" type="vpf_node.py" name="vpf_decode_node" output="screen" respawn="true">
      <param name="topic_name" type="str" value="/fixed/visible/h264"/>
      <param name="rtsp_url" type="str" value="rtsp://admin:123qweasd@192.168.1.68:554/h264/ch1/main/av_stream"/>
  </node>
</launch>
